whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumStatusNotSetSpec.groovy

```diff
@@ -0,0 +1,284 @@
+package net.ripe.db.whois.spec.update
+
+import net.ripe.db.whois.common.IntegrationTest
+import net.ripe.db.whois.spec.BaseQueryUpdateSpec
+import net.ripe.db.whois.spec.domain.AckResponse
+import net.ripe.db.whois.spec.domain.Message
+import spock.lang.Ignore
+
+@org.junit.experimental.categories.Category(IntegrationTest.class)
+class InetnumStatusNotSetSpec extends BaseQueryUpdateSpec {
+
+    // in the database the only parent that exists for NOT-SET resources is ALLOCATED UNSPECIFIED
+    // and the only child resource status with a NOT-SET resource as a parent is LEGACY
+    @Override
+    Map<String, String> getTransients() {
+        [
+                "ALLOC-UNS": """\
+                inetnum:      192.168.0.0 - 192.169.255.255
```

Comment:
Why using the 192.168.0.0 - 192.169.255.255 range as this is part 'private networks' and some not. I would suggest to use 192.168.0.0 - 192.168.255.255 as toplevel.

Yes, I saw it in many other places too.